wok/engine.py

```diff
@@ -346,6 +348,10 @@ def load_pages(self):
 
         # Load files
         for root, dirs, files in os.walk(self.options['content_dir']):
+            # Filter out files if they match any of the ignore patterns
+            for ig in self.options['ignore_files']:
+                files = [ f for f in files if not \
+                        fnmatch.fnmatch(os.path.basename(f), ig) ]
```

Comment:
A minor style nit, the backslash at the end of the line isn't needed. Inside matching brackets (like `()`, `[]`, or `{}`) Python will automatically continue the line. I'd write this more like

```python
files = [f for f in files 
         if not fnmatch.fnmatch(os.path.basename(f), ig)]
```

You could also use `filter` like `files = filter(lambda f: fnmatch(os.path.basename(f), ig), files`.